src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -636,11 +637,39 @@ namespace winrt::TerminalApp::implementation
             LoadSettings();
         }
 
+        UINT dpi = USER_DEFAULT_SCREEN_DPI;
+        int _width = 0;
+        int _height = 0;

```

Comment:
```suggestion
        int desktopWidth = 0;
        int desktopHeight = 0;
```
We generally use `_` to denote that it's a member. So let's remove that to be consistent.

And added `desktop` since you introduced `appWidth` and `appHeight` below. That way there's a better distinction between the two 😉